resources/views/product/partials/gallery/slider.blade.php

```diff
@@ -1,23 +1,33 @@
-<div class="relative" v-if="images.length">
+<div v-if="images.length" class="relative">
     <a
-        :href="config.media_url + '/catalog/product' + images[active]"
-        class="flex items-center justify-center border rounded p-5 h-[440px] cursor-zoom-in"
         v-on:click.prevent="toggleZoom"
+        v-bind:href="config.media_url + '/catalog/product' + images[active]"
+        class="flex h-[440px] cursor-zoom-in items-center justify-center rounded border p-5"
     >
         <img
-            :src="'/storage/{{ config('rapidez.store') }}/resizes/400/magento/catalog/product' + images[active] + '.webp'"
+            src="{{ route('resized-image', [
```

Comment:
The dusk tests are failing because this route name does not exist. Can you check that?